### PR TITLE
Add ckeditor5-html-embed plugin in TextEditor.vue

### PR DESCRIPTION
**Pull request by Zupe007**
I added the ckeditor5-html-embed plugin, so that it is possible to use it in the signature editor.
It enables the use of html signatures.
I made this change, because working with  tables #7282  to create a copy of our company signature doesnt work.

### CODE REVIEW

**Comment by Zupe007** [comment]
@ChristophWurst 
Hey, i hope you are fine. As this is my first pull request i am wondering if i have to do something else to get it reviewed.
Thank you.

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@ckeditor/ckeditor5-theme-lark": "~35.1.0",
     "@ckeditor/ckeditor5-upload": "^35.1.0",
     "@ckeditor/ckeditor5-vue2": "^3.0.1",
+	"@ckeditor/ckeditor5-html-embed": "~35.1.0",
```
Comment (ChristophWurst) [inline package.json:37]: the corresponding package-lock.json change is missing here

Comment (Zupe007) [reply package.json:37]: @ChristophWurst 
I've added the missing entries in the package-lock.json. Sadly i used the wrong account to push this into my repo. I do hope this is not a problem.

Comment (Zupe007) [reply package.json:37]: @ChristophWurst 
Hey there, is everything now to your liking and correct?

Comment (ChristophWurst) [reply package.json:37]: I believe the only thing left to fix is indentation. Look at the file you changed and existing lines before/after. Use the same indentation style.

Comment (Zupe007) [reply package.json]: @ChristophWurst 
I fixed what looked weird to me.
I dont see anything else, if you see something please let me know.

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/mail/pull/7427#issuecomment-1278781552 this looks very strange in the UI to be honest.

**Pull request closed without merging**